cmd/clusterctl/client/delete.go

```diff
@@ -25,8 +25,12 @@ import (
 
 // DeleteOptions carries the options supported by Delete.
 type DeleteOptions struct {
-	// Kubeconfig file to use for accessing the management cluster. If empty, default discovery rules apply.
-	Kubeconfig string
+	// KubeconfigPath defines the path to the kubeconfig file to use for accessing the management cluster. If empty, default discovery rules apply.
+	KubeconfigPath string
```

Comment:
What about 
```suggestion
	Kubeconfig cluster.Config
```
(or the cluster.Config alias)

This is user-facing/breaking, but probably more looking forward than the current approach
@vincepri opinions?

same applies for other Options struct in the client package

Reply:
We relaxed the API guarantees (which we should actually document in a README under clusterctl), so +1

Reply:
I'll map them all to the alias type in the client package and remove the individual properties from the Options across the commands.